src/main.rs

```diff
@@ -68,29 +68,32 @@ fn main() {
 
 /// A Shop is a collection of 3 cards.
 struct Shop {
-    // TOOD: Add the field to this struct
+    cards: [Card; 3],
 }
 
 impl Shop {
     /// Get the price of the most expensive card in the shop
     fn most_expensive(&self) -> u32 {
-        todo!()
+        // There are always three cards, so max is guarenteed to never be none
+        self.cards.iter().map(|c| c.price).max().unwrap()
```

Comment:
An alternative to unwrapping to a panic could be
```rust
self.cards.iter().map(|c| c.price).max().unwrap_or(0)
```
But that said, this is a valid assumption :)